### ISSUE TITLE
A question about dataset

### ISSUE THREAD

**Issue by cy-66** [AfsanaAhmedMunia/AHF-Fusion-U-Net#3]
Hello, I have reproduced your code and found that you only divide the data set into training set and validation set. Where is the test set
